### ISSUE TITLE
Suggestion/Idea: Cli/Terminal only mode 

### ISSUE THREAD

**Issue by lefuturiste** [angryip/ipscan#235]
Hello,

I have been thinking of a cli terminal only mode where no gui show up when you want to scan your network. Do you think this is a great idea ? 
I will be happy to implement this feature myself, but I need the approbation of a maintainer of this repository before I start working of a PR.
If you think this could be implemented, we need to agree on a flag to launch the app as cli only mode.

I will be happy to hear back from you.

**Comment by angryziber** [comment]
Yep, why not. 

Some command-line support is already there, will need to do more console output if hiding the GUI.

On the other hand - it is tricky to support both GUI and cli modes on Windows or Mac.
A second launcher will be needed for that.

**Comment by chrisdlangton** [comment]
just spent my morning trying to "force" it to work in a docker container without x11 and gave up after all the normal tricks failed. i'm not familiar with java/eclipse so i can't contribute.

+1 this 1000x

**Comment by CallumGilly** [comment]
Definitely, would be especially good for piping it into grep to run searches.
